master/pkg/workload/workload.go

```diff
@@ -30,6 +30,11 @@ type Workload struct {
 	TotalBatchesProcessed int  `json:"total_batches_processed"`
 }
 
+// TotalBatches returns the total batch number after the workload is processed.
```

Comment:
I think we just just remove this or call it `TotalBatchesProcessed` and rename `total_batches_processed` to `prior_batches_processed` like it should've been (it makes more sense at total in the harness which is where it originated but master side it's just confusing).

If that's too much work, maybe at least comment `Workload.TotalBatchesProcessed` that it's really prior batches processed

Reply:
> rename total_batches_processed to prior_batches_processed

This will break many places. I'd do that later since I will need to remove this field later.